osu.Game.Rulesets.Tau/Objects/Drawables/DrawableSlider.cs

```diff
@@ -159,6 +172,7 @@ protected override void OnApply()
         {
             base.OnApply();
             path.FadeColour = colour.ForHitResult(HitResult.Great);
+            HighlightHardBeats.BindValueChanged(value => path.Texture = value.NewValue && HitObject.IsHard ? pathTextureHighlighted : pathTexture, true);
```

Comment:
The setting is not dependent on the hitobject, so this does not need to be done on every time this drawable is assigned to a HitObject. This can be done once in `load()`.

Reply:
Disregard, solved the problem by overriding `LoadComplete()`. Binding in `load()` then calling `TriggerChange()` on the bound property allows me to use `renderer` for generating the textures inside the handler while allowing `HitObject` to be populated.